packages/components/src/unit-control/test/index.tsx

```diff
@@ -112,6 +112,7 @@ describe( 'UnitControl', () => {
 			);
 
 			expect(
+				// eslint-disable-next-line testing-library/no-node-access
```

Comment:
There's currently no accessible way to access the wrapper element of the unit control component.